vector/src/main/res/values/strings.xml

```diff
@@ -1031,6 +1037,19 @@
     <string name="room_details_people_invited_group_name">INVITED</string>
     <string name="room_details_people_present_group_name">JOINED</string>
 
+    <!-- Threads -->
+    <string name="room_threads_filter">Filter Threads in room</string>
+    <string name="thread_timeline_title">Thread</string>
+    <string name="thread_list_title">Threads</string>
+    <string name="thread_list_modal_title">Filter</string>
+    <string name="thread_list_modal_all_threads_title">All Threads</string>
+    <string name="thread_list_modal_all_threads_subtitle">Shows all threads from current room</string>
+    <string name="thread_list_modal_my_threads_title">My Threads</string>
+    <string name="thread_list_modal_my_threads_subtitle">Shows all threads you’ve participated in</string>
+    <string name="thread_list_empty_title">Keep discussions organised with threads</string>
+    <string name="thread_list_empty_subtitle">Threads help keep your conversations on-topic and easy to track.</string>
+    <string name="thread_list_empty_notice">Tip: Long tap a message and use “Reply in thread”.</string>
```

Comment:
Maybe better to use %s instead of "Reply in thread" to avoid mistake in translation. You can add a comment above saying that the %s will be replaced by the value of `reply_in_thread`. I see that there is already a diff with the case :)